resources/lib/monitor.py

```diff
@@ -60,6 +61,6 @@ def onNotification(self, sender, method, data):  # pylint: disable=invalid-name
             return
 
         data = utils.decode_data(data)
-        data['id'] = "%s_play_action" % str(sender.replace(".SIGNAL", ""))
+        data['id'] = "%s_play_action" % from_unicode(sender.replace(".SIGNAL", ""))
 
```

Comment:
`"%s_play_action" `is again a unicode literal because you imported `unicode_literals`. Because sender is a variable from ` xbmc.Monitor.onNotification` it makes sense here to decode `sender` to unicode, so use `to_unicode`

Reply:
ok.